test/api/controllers/adminController.test.js

```diff
@@ -99,41 +56,46 @@ describe('Test: admin controller', function () {
 
     it('should add the new collection to the cache', () => {
       this.timeout(50);
+      sandbox.stub(kuzzle.workerListener, 'add').resolves({});
+      sandbox.spy(kuzzle.indexCache, 'add');
+      sandbox.spy(kuzzle.indexCache, 'remove');
+      sandbox.spy(kuzzle.indexCache, 'reset');
 
       return kuzzle.funnel.controllers.admin.updateMapping(requestObject)
         .then(response => {
           should(response).be.instanceof(ResponseObject);
```

Comment:
Did we have already a problem here? missing `done()`? Because we're in asynchronous